### PR TITLE
📖 AMP Geo for URL Rewriting

### PR DESCRIPTION
**Pull request by kristoferbaxter**
Documentation for new feature supporting AMP Geo information in URL rewriting.

### CODE REVIEW

**Comment by kristoferbaxter** [comment]
Friendly ping @samouri and @CrystalOnScript.

File: extensions/amp-link-rewriter/amp-link-rewriter.md [hunk 1]
```diff
@@ -159,21 +159,22 @@ The resulting, rewritten URL would be:
...
-| id           | anchor | `<a href="..." id="link" />`                   | `${id}`             |
-| location     | page   | `https://www.partnerweb.com/`                  | `SOURCE_URL`        |
-| random       | page   | `Math.random().toString(32).substr(2)`         | `${random}`         |
-| referrer     | page   | `https://google.de/`                           | `DOCUMENT_REFERRER` |
-| rel          | anchor | `<a href="..." rel="pass" />`                  | `${rel}`            |
-| rev          | anchor | `<a href="..." rev="author" />`                | `${rev}`            |
-| vars.\*      | config | `{ "vars": { "publisherId": "123" } }`         | `${publisherId}`    |
+| value        | source | example                                        | placeholder           |
+| ------------ | ------ | ---------------------------------------------- | --------------------- |
+| country      | geo    | `us`                                           | `AMP_GEO(ISOCountry)` |
```
Comment (samouri) [inline extensions/amp-link-rewriter/amp-link-rewriter.md:168]: I'm confused by this line -- does the publisher need to include the ISOCountry in the macro?

Comment (kristoferbaxter) [reply extensions/amp-link-rewriter/amp-link-rewriter.md:168]: No they need to indicate ISOCountry as the value (indicates they want the ISOCountry value as the replacement).

Comment (samouri) [reply extensions/amp-link-rewriter/amp-link-rewriter.md:168]: Ah got it. 
Separate: I just checked the unit tests and all the examples yield `country=unknown`

https://github.com/ampproject/amphtml/blob/f564b7ee3cd59972fe86c4edc3052db70852c6f0/test/unit/test-url-replacements.js#L1059-L1115

**Review by samouri — APPROVED** [review]

File: extensions/amp-link-rewriter/amp-link-rewriter.md (hunk 1 above)
Comment (kristoferbaxter) [reply extensions/amp-link-rewriter/amp-link-rewriter.md:168]: Yep, if the value is not known, it's returned as `unknown`. Perhaps we should add tests where the value is resolvable too.

**Pull request merged**